stripe2qbo/sync.py

```diff
@@ -18,10 +18,13 @@
 
 class TransactionSync(BaseModel):
     id: str
+    user_id: str
     created: int
     type: str
     amount: int
     description: str
+    stripe_id: str
+    qbo_account_id: Optional[str] = None
     status: Optional[Literal["pending", "success", "failed"]] = None
     # QBO ids
     transfer_id: Optional[str] = None
```

Comment:
No need to define this twice. Can you delete this and instead import the model defined in db/schemas